Product.md

```diff
@@ -2,32 +2,118 @@
+- I take accountability for the performance of my product area(s), owning the problems to learn how we can improve
+- I seek out learning and inspiration to expand my awareness of different techniques
+- I reflect, regularly, on my personal and professional development to contribute in a growing company
+
+## Senior Product Manager
+
+> I break down silos to make meaningful, regular impact in my product area(s). I use a network to help me find emerging opportunities and spot risks that could impact our work. To bring everyone on the journey I tell stories that create a shared understanding of our customers and how our choices will help them. I build trust in what we do through influence and persuasion, nurturing the team’s reputation for making progress in the toughest of times. I am ready to lead a stream of work with little guidance, developing my strategic and commercial thinking muscles. I share my experiences and learning to help others build their product muscles.
+
+### I build a trusted network that accelerates my ability to find high value opportunities
+
+- I nurture customer and industry relationships who have a track record of providing insights to help us focus on why certain behaviours occur
+- I use structured feedback loops to obtain the insights needed to confirm and/or contradict our understanding of a problem space
+- I watch operations and support work to find opportunities that improve our business efficiency / effectiveness
+- I bring new competitor insights into the organisation to drive activities that maintain our sustainable competitive advantage
+
+### I build advocates for the direction of my product area(s)
+
+- I facilitate debate amongst key stakeholders and influencers on future opportunities to involve them in co-creating the direction for my product area(s)
+- I coordinate product opportunities that extend beyond my area(s) to deliver larger impact to the product
+- I develop a way forward for new or ambiguous situations that are directionally sensible with limited guidance to unlock progress that has been previously out of reach
+- I prune, regularly, my product area(s) to ensure we can deliver a premium experience without overwhelming our users
+
+### I foster an inclusive team that develops a track record for high quality delivery
+
+- I use the hypothesis of our work to test our assumptions and work with my team to discovery what is actually needed
+- I collaborate with the team on risks and failure modes for our work to figure out what we can do to increase our chance of success
+- I include a diverse set of perspectives within and beyond the team on potential solutions to sharpen the team’s thinking about what is essential
+- I can spot when we are blocked or over-analysing to bring us back to small, incremental progress
+- I celebrate wins and share learning throughout the development cycle to build up the team’s mana
```

Comment:
What the heck does this mean?!?

Reply:
Which statement? Here's what I read from it:

Build a risk profile for a piece of work, figure out which ones need mitigation, then do that

Get feedback from others outside of the team on what you are doing

Watch for work that gets blocked or overcooked, then work with the team to simplify/break it down

Promote the good work of the team to build up their reputation for doing good work.